test/users.js

```diff
@@ -0,0 +1,106 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should(),
+  User = require('./../app/models').User,
+  errors = require('./../app/errors'),
+  helperTest = require('./../app/helpers/test');
+
+describe('users', () => {
+  describe('/users POST', () => {
+    it('should fail login because the password is invalid', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({
+          firstName: 'firstName1',
+          lastName: 'lastName1',
+          email: 'email1@wolox.com.ar',
+          password: 'INVALID'
+        })
+        .catch(err => {
+          err.should.have.status(400);
+          err.response.should.be.json;
+          err.response.body.should.have.property('message');
+          err.response.body.should.have.property('internal_code');
+          done();
+        });
+    });
+
+    it('should fail login because the email is invalid and does not belong to the wolox domain', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({
+          firstName: 'firstName1',
```

Comment:
to avoid repeating this objects for testing you can use factory girl as an aid if you want
https://github.com/aexmachina/factory-girl